mentors/javascript/s.json

```diff
@@ -56,4 +56,11 @@
     "avatar_url": null,
     "bio": "JavaScript has been part of my day-to-day job for over 7 years -- A love-hate relationship"
   }
+  {
+    "github_username": "Hakeem-Olasupo",
+    "name": "OLASUPO Abdulhakeem Adigun",
+    "link_text": "digital4us.coder",
+    "link_url": "about.me/aolasupo",
+    "avatar_url": "https://avatars1.githubusercontent.com/u/32235955?s=400&u=df9fef3ee8189d9b1852f5670c2aa090a20935cc&v=4",
+    "bio": "I am Hakeem Olasupo by name, A Blogger, A Graphic designer, A web developer, An Intern at HNG Internship 4.0, a Favcoder, a programmer and Chatbot developer ",
```

Comment:
"bio" should not have a `,` at the end of the line.  Next line after `"bio"` should be `}`.  The last line should be `]` as it currently is.  I recommend using a json inter like https://jsonlint.com/